app/components/m-header.hbs

```diff
@@ -6,30 +6,24 @@
       </Auk::Toolbar::Item>
       <Auk::Toolbar::Item>
         <Auk::Tabs>
-          {{#if this.hasPublicationsEnabled}}
-            <Auk::Tab
-              @route="publications"
-              data-test-m-header-publications
-            >
-              {{t "publications"}}
-            </Auk::Tab>
-          {{/if}}
-          {{#if this.isShownSignatureFolder}}
-            <Auk::Tab
-              @route="signatures"
-              data-test-m-header-signatures
-            >
-              {{t "signature-folder"}}
-            </Auk::Tab>
-          {{/if}}
+          <Auk::Tab
+            @route="search.index"
+            @current-when="search"
+            data-test-m-header-search
+          >
+            <Auk::Icon @name="search" />
+            {{capitalize (t "search")}}
+          </Auk::Tab>
           <Auk::Tab
             @route="agendas.overview"
             @current-when="agendas agenda print-overviews.decisions print-overviews.press-agenda"
             data-test-m-header-agendas
           >
             {{t "agendas"}}
           </Auk::Tab>
-          <Auk::Tab @route="cases" data-test-m-header-cases>{{t "cases"}}</Auk::Tab>
+          <Auk::Tab @route="cases" data-test-m-header-cases>{{t
+              "cases"
+            }}</Auk::Tab>
```

Comment:
is dit prettier ? Ik zou denken dat die `{{t "cases" }}` op 1 lijn zou staan, zoals `{{t "publications"}}`